doc/en/fixture.rst

```diff
@@ -180,8 +180,8 @@ function (in or below the directory where ``conftest.py`` is located)::
     def test_ehlo(smtp):
         response, msg = smtp.ehlo()
         assert response == 250
-        assert "smtp.gmail.com" in str(msg, 'ascii')
```

Comment:
Oh sure, much easier:

```b
assert b"smtp.gmail.com" in msg
```

Will work in py2 and py3. Didn't thought of that at the time. :+1: